<ui version="4.0" >
 <author>RaphaÃ«l Doursenaud</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>725</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>221</red>
        <green>223</green>
        <blue>228</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>85</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>199</red>
        <green>199</green>
        <blue>199</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>103</red>
        <green>141</green>
        <blue>178</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>238</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>82</red>
        <green>24</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>232</red>
        <green>232</green>
        <blue>232</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>221</red>
        <green>223</green>
        <blue>228</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>85</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>199</red>
        <green>199</green>
        <blue>199</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>103</red>
        <green>141</green>
        <blue>178</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>238</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>82</red>
        <green>24</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>232</red>
        <green>232</green>
        <blue>232</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>128</red>
        <green>128</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>221</red>
        <green>223</green>
        <blue>228</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>85</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>199</red>
        <green>199</green>
        <blue>199</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>199</red>
        <green>199</green>
        <blue>199</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>128</red>
        <green>128</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>86</red>
        <green>117</green>
        <blue>148</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>238</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>82</red>
        <green>24</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>232</red>
        <green>232</green>
        <blue>232</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>AutoJet</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="autojet.qrc" >
    <normaloff>:/icons/icons/autojet.svg</normaloff>:/icons/icons/autojet.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" name="gridLayout_2" >
    <item row="1" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font" >
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="tabPosition" >
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <property name="iconSize" >
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="elideMode" >
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="autoMode" >
       <attribute name="title" >
        <string>Auto Mode</string>
       </attribute>
       <attribute name="icon" >
        <iconset resource="autojet.qrc" >
         <normaloff>:/icons/icons/auto.svg</normaloff>:/icons/icons/auto.svg</iconset>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="font" >
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>Don't touch yet</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="manualMode" >
       <attribute name="title" >
        <string>Manual Mode</string>
       </attribute>
       <attribute name="icon" >
        <iconset resource="autojet.qrc" >
         <normaloff>:/icons/icons/manual.svg</normaloff>:/icons/icons/manual.svg</iconset>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="burnersScrollArea" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font" >
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth" >
           <number>0</number>
          </property>
          <property name="horizontalScrollBarPolicy" >
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable" >
           <bool>true</bool>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="burners" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1223</width>
             <height>312</height>
            </rect>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="burnersLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="sizeConstraint" >
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="notepad" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>250</height>
           </size>
          </property>
          <property name="baseSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground" >
           <bool>true</bool>
          </property>
          <property name="title" >
           <string>Notepad</string>
          </property>
          <property name="flat" >
           <bool>false</bool>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="notepadLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QTextEdit" name="notes" >
             <property name="minimumSize" >
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="acceptDrops" >
              <bool>false</bool>
             </property>
             <property name="autoFillBackground" >
              <bool>false</bool>
             </property>
             <property name="frameShape" >
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="lineWidth" >
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="iconSize" >
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionEject" />
   <addaction name="separator" />
   <addaction name="actionCreate_Image" />
   <addaction name="separator" />
   <addaction name="actionBlank" />
   <addaction name="actionSet_Image" />
   <addaction name="actionBurn" />
   <addaction name="separator" />
   <addaction name="actionCancel" />
   <addaction name="separator" />
   <addaction name="actionHelp" />
   <addaction name="actionConfigure" />
   <addaction name="separator" />
   <addaction name="actionShut_Down" />
   <addaction name="actionQuit" />
   <addaction name="separator" />
  </widget>
  <action name="actionConfigure" >
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/configure.svg</normaloff>:/icons/icons/configure.svg</iconset>
   </property>
   <property name="text" >
    <string>Configure (F11)</string>
   </property>
   <property name="toolTip" >
    <string>Configure (F11)</string>
   </property>
   <property name="shortcut" >
    <string>F11</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/exit.svg</normaloff>:/icons/icons/exit.svg</iconset>
   </property>
   <property name="text" >
    <string>Quit (F12)</string>
   </property>
   <property name="toolTip" >
    <string>Quit AutoJet (F12)</string>
   </property>
   <property name="shortcut" >
    <string>F12</string>
   </property>
  </action>
  <action name="actionEject" >
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/eject.svg</normaloff>:/icons/icons/eject.svg</iconset>
   </property>
   <property name="text" >
    <string>Eject/Load (F1)</string>
   </property>
   <property name="toolTip" >
    <string>Eject/Load Trays (F1)</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
  <action name="actionSet_Image" >
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/set_image.svg</normaloff>:/icons/icons/set_image.svg</iconset>
   </property>
   <property name="text" >
    <string>Set Images (F4)</string>
   </property>
   <property name="toolTip" >
    <string>Set Images (F4)</string>
   </property>
   <property name="shortcut" >
    <string>F4</string>
   </property>
  </action>
  <action name="actionBurn" >
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/burn.svg</normaloff>:/icons/icons/burn.svg</iconset>
   </property>
   <property name="text" >
    <string>Burn Discs (F5)</string>
   </property>
   <property name="toolTip" >
    <string>Burn Discs (F5)</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionBlank" >
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/blank.svg</normaloff>:/icons/icons/blank.svg</iconset>
   </property>
   <property name="text" >
    <string>Blank Discs (F3)</string>
   </property>
   <property name="toolTip" >
    <string>Blank Discs (F3)</string>
   </property>
   <property name="shortcut" >
    <string>F3</string>
   </property>
  </action>
  <action name="actionShut_Down" >
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/shutdown.svg</normaloff>:/icons/icons/shutdown.svg</iconset>
   </property>
   <property name="text" >
    <string>Shut Down (F8)</string>
   </property>
   <property name="toolTip" >
    <string>Shut Down DD (F8)</string>
   </property>
   <property name="shortcut" >
    <string>F8</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/help.svg</normaloff>:/icons/icons/help.svg</iconset>
   </property>
   <property name="text" >
    <string>Help (F7)</string>
   </property>
   <property name="iconText" >
    <string>Help (F7)</string>
   </property>
   <property name="toolTip" >
    <string>Help (F7)</string>
   </property>
   <property name="shortcut" >
    <string>F7</string>
   </property>
  </action>
  <action name="actionCreate_Image" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/create_image.svg</normaloff>:/icons/icons/create_image.svg</iconset>
   </property>
   <property name="text" >
    <string>Create Image (F2)</string>
   </property>
   <property name="toolTip" >
    <string>Create Image (F2)</string>
   </property>
   <property name="shortcut" >
    <string>F2</string>
   </property>
  </action>
  <action name="actionCancel" >
   <property name="icon" >
    <iconset resource="autojet.qrc" >
     <normaloff>:/icons/icons/cancel.svg</normaloff>:/icons/icons/cancel.svg</iconset>
   </property>
   <property name="text" >
    <string>Cancel (F6)</string>
   </property>
   <property name="toolTip" >
    <string>Cancel (F6)</string>
   </property>
   <property name="shortcut" >
    <string>F6</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="9" />
 <resources>
  <include location="autojet.qrc" />
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
